### DISCUSSION TITLE
How can i find the libwebkit2gtk-4.0.dll

### DISCUSSION THREAD

**Discussion by mr7bu9ou** [GtkSharp/GtkSharp#376] [Q&A] [answered]
I wonder to known how i can find the libwebkit2gtk-4.0.dll, I searched internet, seems no such binarary.
if anyone own this dll, please share it to me.
 _libraryDefinitions[Library.Webkit] = 
new[] { "libwebkit2gtk-4.0.dll", "libwebkit2gtk-4.0.so.37", "libwebkit2gtk-4.0.dylib", "libwebkit2gtk-4.0.0.dll" };

**Comment by diogob003** [comment, marked as answer]
Those libs refers to [WebKitGtk](https://webkitgtk.org/). That's a huge and complicated thing to deal with.
Which OS are you using?
### macOS
- [MacPorts:](https://ports.macports.org) `sudo port install webkit2-gtk`
- [HomeBrw:](https://brew.sh/) `brew install webkitgtk`
### ubuntu, debian
- `sudo apt install libwebkit2gtk-4.0-37`
- `sudo apt install libwebkit2gtk-4.1-0`
### windows
I do not know, perhaps do not exists.
Some stuff like gtk3 can be found on [MSys2](https://packages.msys2.org/). However I did not found packages for `WebKitGtk`only for `qtWebKit`.

**Comment by lytico** [comment]
> windows I do not know, perhaps do not exists.

it's not supported on windows

see: https://github.com/msys2/MINGW-packages/discussions/9512

**Comment by ghost** [comment]
Is this package now in MSYS2 for gtk 3 or can somebody add it?

**Reply by lytico** [reply]
there is no libwebkit on windows

**Reply by ghost** [reply]
Why can't it be compiled for windows? 

Does there exist another widget for GTK3 that is doing the same and available for windows?

**Reply by lytico** [reply]
ask upstream: https://github.com/msys2/MINGW-packages/discussions/9512
